packages/babel-parser/test/fixtures/experimental/import-module-attributes/invalid-syntax-with-no-attributes-identifier/input.js

```diff
@@ -0,0 +1,2 @@
+import "x"
+with ({});
```

Comment:
@xtuc Although `with` statement is forbidden in modules, do we allow LineTerminator before `WithClause`? I notice that`AsClause` does forbid leading line terminators: https://tc39.es/proposal-module-attributes/#prod-AsClause

The current implementation allow the following to be parsed.
```js
import "x"
with type: "jpeg"
```